workflow/emc-global/scripts/exglobal_prep_chem.sh

```diff
@@ -152,7 +124,7 @@ else
     export GBBEPX_DATA_DIR=$GBBEPX_DATA_DIR_TODAY    
     export gbbepx_days=${gbbepx_days:-'$PDYm1'}
 fi
-gbbepx_pattern=${gbbepx_pattern:-'$GBBEPX_DATA_DIR/${local_name}.003.${day}.FV3.${CASE}Grid.${tiledir}.bin'}
+gbbepx_pattern=${gbbepx_pattern:-'$GBBEPX_DATA_DIR/${local_name}-${CASE}${GTtile}_v4r0_blend_s${day}'}
```

Comment:
Why does the GTtile was set value in the after the lines? Such as Line 138 and 206?

Reply:
Thanks I understood now.